docs/deployement/RaspberryDeploy.md

```diff
@@ -26,29 +26,30 @@ Now you need to configure the ufw to allow the ports that you want to use on Lio
 Please keep in mind that you need to allow the port that you will use for LionkApp and SSH.
 
 ```bash
-sudo ufw allow 22
+sudo ufw allow 22 # 22 or the port that you use for SSH
 sudo ufw allow <LionkAppPort>
 sudo ufw enable
 ```
 
 ## 4. Create a docker-compose.yml file
 
-Create a docker-compose.yml file in the directory that you want to deploy LionkApp. You can use the following template for the docker-compose.yml file.
+Create a docker-compose.yml file in the directory that you want to deploy LionkApp. You can use the following template for the docker-compose.yml file or copy the file in [../src/docker-compose.yml](../src/docker-compose.yml)
```

Comment:
This part, "or copy the file in ......" must be removed because we don't have the docker-compose.yml anymore